scripts/pre_push_hook.py

```diff
@@ -373,12 +391,19 @@ def main(args=None):
                         'Push failed, please correct the linting issues above.')
                     sys.exit(1)
             frontend_status = 0
+            travis_ci_check_status = 0
             if does_diff_include_js_or_ts_files(files_to_lint):
                 frontend_status = start_python_script(FRONTEND_TEST_SCRIPT)
             if frontend_status != 0:
                 python_utils.PRINT(
                     'Push aborted due to failing frontend tests.')
                 sys.exit(1)
+            if does_diff_include_travis_yaml_or_protractor_conf_file(files_to_lint):
+                travis_ci_check_status = start_python_script(TRAVIS_CI_PROTRACTOR_CHECK_SCRIPT)
+            if travis_ci_check_status != 0:
+                python_utils.PRINT(
+                    'Push aborted due to failing travis ci checks.')
```

Comment:
Try and make this sentence clearer: it is due to checks failing for the e2e test configuration. The current message reads as though it's a problem with the Travis-CI results.

Reply:
Done